src/plugins/sqlalchemy_create_object_operator.py

```diff
@@ -104,7 +136,7 @@ def execute(self, context: Dict[str, Any]) -> None:
         else:
             self.data_table_name = self.data_table_name
 
-        data_schema_url = SCHEMA_URL / self.data_schema_name / self.data_schema_name
+        data_schema_url = self.get_schema(self.data_schema_version)
```

Comment:
If you implement https://github.com/Amsterdam/dataservices-airflow/pull/253/files#r635806753
this becomes:

```python
data_schema_url = self.get_schema()
```

Reply:
:+1: